src/components/Trade/TradeInput.tsx

```diff
@@ -74,7 +74,9 @@ export const TradeInput = ({ history }: RouterProps) => {
       : selectAssetByCAIP19(state, 'eip155:1/slip44:60'),
   )
   const feeAssetBalance = useAppSelector(state =>
-    feeAsset ? selectPortfolioCryptoHumanBalanceByAssetId(state, feeAsset?.caip19) : null,
+    feeAsset
+      ? selectPortfolioCryptoHumanBalanceByAssetId(state, { assetId: feeAsset?.caip19 })
```

Comment:
why are we using an object instead of just passing the value?

Reply:
We needed to change the arity, because it was previously different in the staking data slice - we also need different arguments at possibly different positions, which is going to be an issue when using the same selectors. Using an object solves all of our concerns without having to do some flaky arity dance that could potentially break things.